### PR TITLE
Add OverblogGraphqlBundle recipe

### PR DESCRIPTION
**Pull request by renatomefi**
| Q             | A
| ------------- | ---
| License       | MIT

How to test:
- `composer create-project "symfony/skeleton^3.3" sf-flex-graphql-recipe && cd sf-flex-graphql-recipe`
- `SYMFONY_ENDPOINT=https://symfony.sh/r/github.com/symfony/recipes-contrib/85 composer req cli:^3.3 webserver:^3.3 twig:^3.3 overblog/graphql-bundle:0.9.x-dev`
- `bin/console server:run`
- http://127.0.0.1:8000/graphiql

### CODE REVIEW

**Comment by renatomefi** [comment]
Just starting over a new PR since I was getting some bugs with the deployment tool:
Previous PR is https://github.com/symfony/recipes-contrib/pull/85

File: overblog/graphql-bundle/0.9/config/graphql/types/query.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Query:
+    type: object
+    config:
+        description: A Hello World query.
+        fields:
+            hello:
+                type: String
+                resolve: "world"
```
Comment (fabpot) [inline overblog/graphql-bundle/0.9/config/graphql/types/query.yaml]: Should probably be removed as this is an example.

Comment (renatomefi) [reply overblog/graphql-bundle/0.9/config/graphql/types/query.yaml]: @fabpot removed, we only now point the user to the docs

**Comment by renatomefi** [comment]
@fabpot @stof could you please merge it? thank you!

**Pull request merged**